test/test_gpu.jl

```diff
@@ -1,6 +1,10 @@
 # This file requires a GPU in order to run. It gets tested as part of the
 # JuliaGPU CI on Gitlab. Contact @odow for more details.
 
+# CUDA_jll is not in our Project.toml, so we need to install it on GITLAB_CI.
+import Pkg
+Pkg.add("CUDA_jll")
```

Comment:
```suggestion
pkg"add CUDA_jll@9.0"
```
I actually don't known how to use `PackageSpec` with `Pkg.add` to achieve this ;)
`SCS_GPU` was built against this version:

https://github.com/JuliaPackaging/Yggdrasil/blob/01037ee342dd40ea57c3ad20e69ad8bc81d24bbe/S/SCS_GPU/build_tarballs.jl#L38

This was an arbitrary choice, I picked the lowest available to support possibly widest range of GPUs